stable/sensu/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
         args:
           - api        
 {{ toYaml .Values.api.resources | indent 8 }}          
```

Comment:
It looks like it is there but doesn't have the key. It also looks like the server resources in broken too. Can you fix that as well and yank this line?